package.json

```diff
@@ -4,10 +4,11 @@
   "description": "Experimental speadsheet-based interface for structured programming based on \"structured grammars\".",
   "main": "dist/main.js",
   "scripts": {
+    "dev": "watch 'npm run start' src",
     "test": "echo \"Error: no test specified\" && exit 1",
     "start-cargo": "cargo watch -w src/ -s './scripts/build.sh && electron .'",
     "start": "webpack --config ./webpack.config.js && electron dist/index.html",
-    "watch": "cargo watch -w src/ -w static/styles.css -s 'npm run start'"
+    "watch": "cargo watch -w src/ -w static/styles.css -s 'npm run build'"
```

Comment:
can you explain why you made this change?

Reply:
It’s for VS code on window that run watch for every change in src folder. After install npm watch. Then run with npm run dev

Reply:
It’s for VS code on window that run watch for every change in src folder. After install npm watch. Then run with npm run dev

Reply:
Actually, I forgot why it's changed there, but I see it's nothing different from start and build; so I change back to 'npm run start'